tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs

```diff
@@ -19,9 +17,9 @@ public class SAPXmlHelper
 
         static int actionCounter;
 
-        static List<string> AttrNotMatched = new List<string>();
+        static readonly List<string> AttrNotMatched = new List<string>();
         static List<string> ChangeAVCSUoS = new List<string>();
-        static Dictionary<string, List<string>> ChangeENCCell = new Dictionary<string, List<string>>();
```

Comment:
Can be simplified as 
static readonly Dictionary<string, List<string>> ChangeENCCell = new();

Reply:
Updated